packages/llvm21_dev.rb

```diff
@@ -34,6 +34,12 @@ class Llvm21_dev < Package
   no_source_build
   no_strip
 
+  def self.preflight
+    abort "Update #{CREW_LLVM_VER}_build first.".lightred if Gem::Version.new(version.split('-').first) < Gem::Version.new(Llvm21_build.version.split('-').first)
+    llvm_lib_obj = Package.load_package("#{__dir__}/#{CREW_LLVM_VER}_dev.rb")
+    abort "Update  #{CREW_LLVM_VER}_lib first.".lightred if Gem::Version.new(version.split('-').first) > Gem::Version.new(llvm_lib_obj.version.split('-').first)
```

Comment:
Would it make sense to display both messages and then abort rather than updating the build and then seeing the second message afterwards?